tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -762,6 +766,11 @@ def test_custom_metrics_cardinality_limit(self):
                 }
             }
         ]
+
+        limit = config[0]["limit"]
+        normalized_limit = normalize_cardinality_limit_config(limit)
+        assert normalized_limit == limit
```

Comment:
If normalize_cardinality_limit_config does not change anything, and we assert that they are the same before and after, why do we even call it? If we know it doesn't do anything, why do we assert?